draft-ietf-rtcweb-security-arch.xml

```diff
@@ -1722,12 +1722,25 @@ a=sendrecv
                   </list>
                 </t>
                 <t>
-                  Sites that have identities that do not fit into the RFC822
-                  style (for instance, identifiers that are simple numeric
-                  values, or values that contain '@' characters) SHOULD convert
-                  them to this form by escaping illegal characters and appending
-                  their IdP domain (e.g., user%40133@identity.example.com), thus
-                  ensuring that they are authoritative for the identity.
+                  Sites that use identities that are not of the form
+                  &lt;user&gt;@&lt;domain&gt;, or which use identities in
+                  which the domain portion does not match the domain name of
```

Comment:
I would not support Harald's observation (which would be reasonable if this were made 4 years ago), but would instead suggest that this simply say that if the user portion of the identity contains the characters '%' or '@', then those characters MUST be escaped [...].

Reply:
I started out trying to say that, but had difficult saying it in a sufficiently precise way. I'm happy to take edits if you think you can make a run at it, but I tried and wasn't satisfied with the result.

Reply:
see https://github.com/adamroach/security-arch/pull/1